tests/data/firefox/tinderbox-builds/mozilla-central-win32/1425327271/firefox-25.0a1.en-US.win32.txt

```diff
@@ -0,0 +1,2 @@
+20150302121431
+https://hg.mozilla.org/mozilla-central/rev/abb7f0d180da
```

Comment:
Please do not create new folders. You can put this txt file into an existing one. I would propose always the first one under the platform like https://github.com/mozilla/mozdownload/tree/master/tests/data/firefox/tinderbox-builds/mozilla-central-win32/1374568307.